app/models/concerns/smooch_messages.rb

```diff
@@ -430,6 +430,7 @@ def smoooch_post_save_message_actions(message, associated, app_id, author, reque
     end
 
     def smooch_relate_items_for_same_message(message, associated, app_id, author, request_type, associated_obj)
+      return if associated.class.name != 'ProjectMedia'
```

Comment:
@caiosba then it's best to change this to `return unless associated_obj.is_a?(ProjectMedia)`

Reply:
Yes, it's more Ruby-ish and easier to read :)

That's something I would say when I review :D